### ISSUE TITLE
Worked fine for a few days..

### ISSUE THREAD

**Issue by LostSoulFly** [AuroraWright/SafeA9LHInstaller#23]
[arm9.zip](https://github.com/AuroraWright/SafeA9LHInstaller/files/514913/arm9.zip)

MY N3DS has worked wonderfully for months up to this point. A few days ago, I did the OTPLess install with SafeA9LHInstaller. Things worked great for a few days. I restored my emuNAND into my sysNAND and everything was wonderful. A day or two later I went to launch a game and as it was booting, both screens became garbled and I had to power it off. It came back up and worked fine, though.. However, while I was playing a game a day or two after that, my N3DS froze completely. I had to power it down. It hasn't worked correctly since.

I've only gotten my N3DS to boot 1 or two times out of 30+ in the last three days. It turns on, the blue light comes on, and nothing else. I either have to remove the battery or hold the power button.

If I remove my SD card and power it on, the light comes on and goes off quickly, indicating that it can't find the files on the SD card. To be sure, I took my original N3DS's MicroSD and formatted and set it up exactly according to Plailect's guide (again, including formatting with the GUI utility at 32K sector). Nothing changed. The power light stays solid blue until forced off by power button or removing battery.

The strangest thing is that every so often it will actually turn on. Sometimes it lets me get into Hourglass9 or Luma's config. (It hasn't done this in over 24 hours, though, so I'm not sure if it will ever again). It doesn't matter which MicroSD is inserted. I even tried my wife's Micro SD, which is setup and working OTPLess currently. Hourglass9 loaded twice yesterday, showed some strings of text as it loaded, and then the text disappeared but the logo remained on the top screen. I had to power it off

I've attached two memory dumps from the Luma folder. I'm open to any ideas and desperately need help.

If I can get it to load again, I'll try to restore my original sysNAND and remove a9lh to see if it changes anything, but at this point i'm not hopeful it will come back on..

Edit: I'm in the process of ordering a used N3DS XL along with some appropriate tools (#000 phillips) to disassemble my old one and the one currently en route. I'm going to swap the motherboards and hopefully bring my old one back to life.. it has dual IPS displays :(

**Comment by AuroraWright** [comment]
This has nothing to do with SafeA9LHInstaller since A9LH ran fine, looks like an hardware issue...

**Comment by LostSoulFly** [comment]
It's moot at this point, but I assumed it would be of interest since my 3DS could recognize when an SD card with the A9LH bin was inserted. This lead me to believe that the issue was somehow with the installation or reliability of the methods used.

**Comment by AuroraWright** [comment]
It's not

**Comment by TuxSH** [comment]
> both screens became garbled and I had to power it off (etc.)
This is cleary an hardware issue, and as nothing to do with any exploit installation or software at all.

**Comment by LostSoulFly** [comment]
>This is cleary an hardware issue, and as nothing to do with any exploit installation or software at all.

Yeah, that's certainly a valid assumption, but after I had done the hardmod it came back up with the blue bootrom screen. At that point the 3DS appeared to work, it just didn't want to boot through A9LH.

**Comment by Mrrraou** [comment]
Still has nothing to do with the software. This is an hardware issue or you are likely faking at that point.

**Comment by AuroraWright** [comment]
The fact it shows the bluescreen after an hardmod doesn't show anything (as the hardmod just makes bootrom error out when reading the NAND when it's connected externally). As an example, the SD slot might be malfunctioning and this would cause A9LH to fail.

**Comment by LostSoulFly** [comment]
I still think it was a software issue. Perhaps not entirely, and perhaps it's completely circumstantial that it happened after OTPless A9LH despite it working flawlessly for months. You can take my plight as a fake or a joke, or you can look at it objectively as someone that was in need of help and never got it.

At the end of the day, I messed some of the small black clips and wasn't able to finish the repairs because I couldn't get the cables to be seated properly again. However, I was able to get it to boot to the bootrom screen and show up on my computer. I was in the 3DSHacks discord and an awesomely helpful person named Pbanj (and a few others) did what they could to help troubleshoot. He was convinced that a hardmod would solve the issue. And as I said, it booted to the blue bootrom screen.

I have no definitive proof one way or the other and it ultimately doesn't matter if you believe me, but I thought I would ask for help at the best place I could think of and maybe save other people from a similar issue in the future.

>As an example, the SD slot might be malfunctioning and this would cause A9LH to fail.

I tried two separate MicroSD readers from two separate N3DS. It was likely not the reader (that was my first thought as well). The 3DS could tell when the an SD card was inserted and had the bin file on it. The same SD card without the file would cause the 3DS to turn on and then off as if there was no SD card. The screens didn't initialize on boot until I did the hardmod, then I got the bluescreen.

**Comment by AuroraWright** [comment]
Are you aware that the A9LH binaries are used by _hundreds_ of people and they work for everyone? Plus:
"I went to launch a game and as it was booting, both screens became garbled and I had to power it off."

"while I was playing a game a day or two after that, my N3DS froze completely. I had to power it down. It hasn't worked correctly since."

This has **nothing** to do whatsoever, with A9LH. So it's strongly suggested that you have an hardware issue of some sort.
